cli/dts/lib.deno.shared_globals.d.ts

```diff
@@ -658,6 +658,25 @@ interface WorkerEventMap extends AbstractWorkerEventMap {
   "messageerror": MessageEvent;
 }
 
+/**
+ * "inherit" will take the permissions of the thread the worker is created in
+ */
+type WorkerPermissionBooleanOption = "inherit" | boolean;
+
+/**
+ * "inherit" will take the permissions of the thread the worker is created in
+ * You can provide a list of routes relative to the file the worker is created in
+ * to limit the access of the worker
+ */
+type WorkerPermissionListOption = "inherit" | boolean | Array<string>;
+
+/**
+ * "inherit" will take the permissions of the thread the worker is created in
+ * You can provide a list of routes relative to the file the worker is created in
+ * to limit the access of the worker
+ */
+type WorkerPermissionPathOption = "inherit" | boolean | Array<string | URL>;
```

Comment:
I think these should be inlined. The names and implied categorisations (boolean vs list vs path) are a bit awkward and don't properly represent the relationship between path allowlists and other types of allowlists. It's much better to document these per permission type. It also makes the docs easier to follow.

Reply:
Ok, I'll add the docs above to the worker jsdoc then